app/resources/status.html.template

```diff
@@ -0,0 +1,83 @@
+{# Copyright 2010 Google Inc.  Licensed under the Apache License, Version   #}
```

Comment:
BTW, you can find PFIF spec at the following site. Person Finder is a reference implementation of the spec.

http://zesty.ca/pfif/1.4/